opentelemetry-instrumentation/README.rst

```diff
@@ -71,13 +72,14 @@ Well known trace exporter names:
     - jaeger_proto
     - jaeger_thrift
     - opencensus
-    - otlp
-    - otlp_proto_grpc
-    - otlp_proto_http
     - zipkin_json
     - zipkin_proto
+    - otlp
+    - otlp_proto_grpc (`deprecated`)
+    - otlp_proto_http (`deprecated`)
 
-``otlp`` is an alias for ``otlp_proto_grpc``.
+Note: The default transport protocol for ``otlp`` is gRPC.
+HTTP is currently supported for traces only, and should be set using ``OTEL_EXPORTER_OTLP_TRACES_PROTOCOL=http/protobuf``
```

Comment:
I suggest not using the more general way of `OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf`, since it's still not supported for metrics & logs.